clowder-framework/clowder2
# Frontend refreshing mechanism

To test
- set keycloak access token lifespan to 1 minute
<img width="712" alt="image" src="https://user-images.githubusercontent.com/13950475/167028710-4c43c4db-b3a6-4d74-8545-a9468cd46de6.png">
- watch the network tab to see refresh and retry 
<img width="731" alt="image" src="https://user-images.githubusercontent.com/13950475/167028802-ee3b8eff-1c58-468f-abfa-e588cef74a06.png">
